services/publicapi/run_transaction.go

```diff
@@ -4,28 +4,32 @@ import (
 	"context"
 	"github.com/orbs-network/orbs-network-go/crypto/digest"
 	"github.com/orbs-network/orbs-network-go/instrumentation/log"
+	"github.com/orbs-network/orbs-network-go/instrumentation/trace"
 	"github.com/orbs-network/orbs-spec/types/go/protocol"
 	"github.com/orbs-network/orbs-spec/types/go/protocol/client"
 	"github.com/orbs-network/orbs-spec/types/go/services"
 	"github.com/pkg/errors"
 	"time"
 )
 
-func (s *service) CallMethod(ctx context.Context, input *services.CallMethodInput) (*services.CallMethodOutput, error) {
+func (s *service) CallMethod(parentCtx context.Context, input *services.CallMethodInput) (*services.CallMethodOutput, error) {
 	if input.ClientRequest == nil {
 		err := errors.Errorf("error: missing input (client request is nil)")
 		s.logger.Info("call method received via public api failed", log.Error(err))
 		return nil, err
 	}
 
+	ctx := trace.NewContext(parentCtx, "PublicApi.SendTransaction")
```

Comment:
this is call method why is the context here send transaction ?

Reply:
copy-paste :(